tests/actions/ReportTest.js

```diff
@@ -430,40 +430,57 @@ describe('actions/Report', () => {
         // We should generate link
         let originalCommentHTML = 'Original Comment';
         let afterEditCommentText = 'Original Comment www.google.com';
-        let newCommentMarkdown = Report.handleUserDeletedLinks(afterEditCommentText, originalCommentHTML);
-        let expectedOutput = 'Original Comment [www.google.com](https://www.google.com)';
+        let newCommentMarkdown = Report.handleUserDeletedLinksInHtml(afterEditCommentText, originalCommentHTML);
+        let expectedOutput = 'Original Comment <a href="https://www.google.com" target="_blank" rel="noreferrer noopener">www.google.com</a>';
         expect(newCommentMarkdown).toBe(expectedOutput);
 
         // User deletes www.google.com link from comment but keeps link text
         // We should not generate link
         originalCommentHTML = 'Comment <a href="https://www.google.com" target="_blank">www.google.com</a>';
         afterEditCommentText = 'Comment www.google.com';
-        newCommentMarkdown = Report.handleUserDeletedLinks(afterEditCommentText, originalCommentHTML);
+        newCommentMarkdown = Report.handleUserDeletedLinksInHtml(afterEditCommentText, originalCommentHTML);
         expectedOutput = 'Comment www.google.com';
         expect(newCommentMarkdown).toBe(expectedOutput);
 
         // User Delete only () part of link but leaves the []
         // We should not generate link
         originalCommentHTML = 'Comment <a href="https://www.google.com" target="_blank">www.google.com</a>';
         afterEditCommentText = 'Comment [www.google.com]';
-        newCommentMarkdown = Report.handleUserDeletedLinks(afterEditCommentText, originalCommentHTML);
+        newCommentMarkdown = Report.handleUserDeletedLinksInHtml(afterEditCommentText, originalCommentHTML);
```

Comment:
Should we change the name `newCommentMarkdown` here to `newCommentHtml`

Reply:
updated